src/test/java/com/microsoft/sqlserver/jdbc/bulkCopy/BulkCopyISQLServerBulkRecordTest.java

```diff
@@ -118,6 +143,39 @@ public void testISQLServerBulkRecord() throws SQLException {
                 data.add(CurrentRow);
             }
         }
+        
+        BulkData(DBTable dstTable, boolean dtTest) {

```

Comment:
Can we update the existing BulkData() to support the LocalDataTime test? And the `dtTest` param is not used. I see that it is used to distinguish it from existing BulkData() but there should be a better way to handle this.